packages/wrangler/src/reporting.ts

```diff
@@ -72,7 +80,7 @@ export async function reportError(err: Error, origin = "") {
       message: "Would you like to submit a report when an error occurs?",
       choices: [
         { title: "Always", value: true },
-        { title: "Yes", value: "once" },
+        { title: "Only this time", value: "once" },
```

Comment:
"Only this time" is a little misleading because, to me, it suggests that we will do it this once and then never ask again...